openvidu-server/src/test/resources/integration-test.properties

```diff
@@ -38,6 +38,7 @@ OPENVIDU_STREAMS_VIDEO_MAX_SEND_BANDWIDTH=1000
 OPENVIDU_STREAMS_VIDEO_MIN_SEND_BANDWIDTH=300
 OPENVIDU_STREAMS_FORCED_VIDEO_CODEC=VP8
 OPENVIDU_STREAMS_ALLOW_TRANSCODING=false
+OPENVIDU_STREAMS_ALLOW_SIMULCAST=false
```

Comment:
Here set as `false` because tests prior to this change were assuming non-simulcast... we might want to write new tests for simulcast. But, for now, setting this to false here is the correct thing to do, right?